src/app/technical-record/test-history/test-history.component.ts

```diff
@@ -1,34 +1,16 @@
-import {Component, OnInit, ChangeDetectionStrategy, Input} from '@angular/core';
+import { Component, OnInit, ChangeDetectionStrategy, Input } from '@angular/core';
 import { TestResultModel } from '@app/models/test-result.model';
 
 @Component({
   selector: 'vtm-test-history',
   templateUrl: './test-history.component.html',
+  styleUrls: ['../../app.component.scss'],
```

Comment:
What do we need this file for?

Reply:
This is technical record, I only took care on test record files cleanup. I don't mind removing if needed.